dashboard/templates/dashboard/index.html

```diff
@@ -65,12 +67,21 @@
                 </div>
             </nav>
         {% elif user_type == constant_ut_adminStaff %}
-            <nav class="nav d-flex">
-                <a class="p-2 text-muted active" href="{% url 'dashboard:admissions:index' %}"
-                   data-toggle="tab">Student
-                    Applications</a>
-                <a class="p-2 text-muted" href="#school_informations" data-toggle="tab">School
-                    Informations</a>
+            <nav class="nav d-flex justify-content-between">
+                <div class="nav nav-tabs d-flex justify-content-between" id="dashboard-tab"
+                     role="tablist" style="width: 100%;">
+                    {# TODO: update this    #}
```

Comment:
I'm a bit confused about this todo, it'll be helpful if you can elaborate it a bit more.

Reply:
This Todo was already present for student tab. Since I was supposed to replicate the same for admin staff, I let the comment be the same too. Not sure who added it for student or what it means.